Source/monster.h

```diff
@@ -307,9 +307,9 @@ struct Monster { // note: missing field _mAFNum
 		return pgettext("monster", data().name);
 	}
 
-	int exp(_difficulty difficulty) const
+	uint32_t exp(_difficulty difficulty) const
 	{
-		int monsterExp = data().exp;
+		unit32_t monsterExp = data().exp;

```

Comment:
`unsigned` is fine, this doesn't need to be a fixed width field. It'll eventually get added to `Player::experience` which is `uint32_t` but that's after a division and cap has been applied, and the fixed width type there is because it's a persistent data structure. For function return values it's not critical to specify the size.